### ISSUE TITLE
Visualization problem with native-ui

### ISSUE THREAD

**Issue by pietrodn** [platformio/platformio-atom-ide-terminal#166]
The buttons of the terminal plugin overlap with the name of the currently opened file with the native-ui theme (bottom left edge).
![schermata 2017-02-12 alle 12 01 59](https://cloud.githubusercontent.com/assets/5058646/22861530/2e8cb720-f11b-11e6-8dad-81b6811d7791.png)

**Comment by shixukai** [comment]
+1

**Comment by sectsect** [comment]
## First aid

Add the following code to your `styles.less`.
```less
// Fix bug for platformio-atom-ide-terminal  v2.2.3.
.platformio-ide-terminal.status-bar {
	position: relative;
	vertical-align: middle;
}
```

**Comment by pietrodn** [comment]
Thanks, now the buttons look fine again!
![schermata 2017-02-14 alle 10 41 01](https://cloud.githubusercontent.com/assets/5058646/22923517/1c7e1a5a-f2a2-11e6-83e3-a4a0523a3c37.png)
